<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>769</width>
    <height>569</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial Black</family>
    <underline>false</underline>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabs">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>0</y>
      <width>641</width>
      <height>551</height>
     </rect>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="Files">
     <attribute name="title">
      <string>Files</string>
     </attribute>
     <widget class="QListWidget" name="filesList">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>51</y>
        <width>631</width>
        <height>371</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="filesHeader">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>591</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>FILE SOURCE : NONE</string>
      </property>
     </widget>
     <widget class="QPushButton" name="renderBtn">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>440</y>
        <width>93</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>SHOW FILE</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="itemsCheck">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>450</y>
        <width>131</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>SHOW ALL</string>
      </property>
     </widget>
     <widget class="QLabel" name="itemsInfo">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>440</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>TOTAL ITEMS : 0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="open_raw">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>450</y>
        <width>93</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Open File</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="FileView">
     <attribute name="title">
      <string>TEXT</string>
     </attribute>
     <widget class="QLineEdit" name="front_file_name">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>30</y>
        <width>271</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="content">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>90</y>
        <width>621</width>
        <height>291</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Sans Serif Collection</family>
        <pointsize>10</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="text">
       <string>File Name :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="save_file">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>SAVE File</string>
      </property>
     </widget>
     <widget class="QPushButton" name="copy_cont">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>410</y>
        <width>91</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy </string>
      </property>
     </widget>
     <widget class="QPushButton" name="show_enlarged">
      <property name="geometry">
       <rect>
        <x>502</x>
        <y>390</y>
        <width>131</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Enlarge</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear_btn">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>410</y>
        <width>93</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string>CLEAR</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>-10</x>
        <y>70</y>
        <width>711</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="settings">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <attribute name="title">
      <string>settings</string>
     </attribute>
     <widget class="QLabel" name="header1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>271</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Path to which scripts will be saved  :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="path_s">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>70</y>
        <width>461</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="browse_path">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>70</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QLabel" name="header3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>(assets path )</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="header2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>70</y>
        <width>113</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>(Path_scripts)</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="path_a">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>150</y>
        <width>421</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QToolButton" name="browse_assets">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>150</y>
        <width>31</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="defBtn">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>210</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>To Default</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>310</y>
        <width>461</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>** Just Reopen the application after you just changed any path </string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
